tools/run_tests/xds_k8s_test_driver/framework/test_app/client_app.py

```diff
@@ -312,6 +413,24 @@ def find_subchannels_with_state(
                     subchannels.append(subchannel)
         return subchannels
 
+    def check_channel_active(
+        self, channel: _ChannelzChannel, **kwargs
+    ) -> _ChannelzChannel:
+        """Checks if the channel is active.
+
+        We consider the channel is active if channel is in READY state and calls_started is
+        greater than calls_failed.
+        """
+        if (
+            channel.data.state.state is _ChannelzChannelState.READY
+            and channel.data.calls_started > channel.data.calls_failed
+        ):
+            return channel
+
+        raise self.NotFound(
+            f"[{self.hostname}] Not found an active XDS channel."
```

Comment:
```suggestion
            f"[{self.hostname}] Not found successful calls over the channel."
```
because check_channel_active is not xDS channel soecific